docs/source/configuration/overview.mdx

```diff
@@ -188,6 +188,31 @@ server:
 
 ```
 
+### Introspection
+
+By default, the router answers to some introspection queries. You can override this behavior to disable the introspection like so:
+
+```yaml title="router.yaml"
+#
+# server: Configuration of the HTTP server
+#
+server:
+  introspection: false
+```
+
```

Comment:
Is this intruding from a different PR?

Reply:
It was introduced in this PR https://github.com/apollographql/router/pull/749 but not documented